### PR TITLE
Multi-license under MIT OR Apache-2.0 OR BSD-1-Clause

### PR DESCRIPTION
**Pull request by JasonGross**
Hopefully this will satisfy the Go team and might come closer to getting
us into OpenSSL.

We follow
https://github.com/rust-lang/rust/tree/e0bc267512fc0cb49c86978192857e8187017f0b#license
and
https://github.com/rust-lang/rust/blob/e0bc267512fc0cb49c86978192857e8187017f0b/COPYRIGHT
for wording.

Got out-of-band approval from all the people (and representatives of all the companies) in `AUTHORS`.  Please review for whether the way I'm laying things out seems good.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,6 +141,14 @@ Here are some examples of ways to invoke the binaries (from the directories that
 
 You can find more examples in the `Makefile`.
 
+License
+-------
+
+Fiat-Crypto is distrubted under the terms of both the MIT License, the Apache License (Version 2.0), and the BSD 1-Clause License; users may pick which license to apply.
```
Comment (mdempsky) [inline README.md:147]: nit: typo: "disturbted" should be "distributed"

Also maybe omit "both"? Merriam Webster does admit using "both" for more than two things. But it sounds funny to me, and the sentence reads fine without it, IMO. (If it's removed, it should be removed from fiat-rust/README.md too.)

Comment (JasonGross) [reply README.md]: Thanks, updated

**Comment by achlipala** [comment]
Looks fine, relative to my meager knowledge of the subject!

**Review by agl — APPROVED** [review]

**Comment by agl** [comment]
Approved on behalf of Google to relicense as all of MIT, Apache-2.0, and BSD-1-Clause.

**Comment by davidben** [comment]
(Removed myself from the reviewer list since AGL has responded.)

**Review by andres-erbsen — APPROVED** [review]

**Review by jadephilipoom — APPROVED** [review]

**Review by achlipala — APPROVED** [review]

**Comment by paulidale** [comment]
Thanks for making the effort to do this.  OpenSSL is looking into ways to move to using this and ECC-Kila as an option.

**Pull request merged**